Leetcode-1247/Solution.cpp

```diff
@@ -0,0 +1,43 @@
+// Clarity Questions -
+// Are there only 2 characters in the string? (Just to confirm)
+// What is the max length of strings?
+// Is there any limit on the number index-wise unequal elements in both the strings? (elements in different strings with same index but different values)
+// Is there any relation between number of Xs and number of Ys?
```

Comment:
not good questions, it seems like u are asking for asking sake,
1. max length makes sense (because that might effect the time complexity)